diff -ruN MG5_aMC_v2_6_5_orig/Template/LO/SubProcesses/setcuts.f MG5_aMC_v2_6_5/Template/LO/SubProcesses/setcuts.f
--- MG5_aMC_v2_6_5_orig/Template/LO/SubProcesses/setcuts.f	2019-02-04 05:19:42.000000000 -0600
+++ MG5_aMC_v2_6_5/Template/LO/SubProcesses/setcuts.f	2022-09-08 16:40:35.933463699 -0500
@@ -221,6 +221,7 @@
          endif
 
          if (abs(idup(i,1,iproc)).eq.21)  is_a_j(i)=.true. ! gluon is a jet
+$disableDarkJetPatch         if (abs(idup(i,1,iproc)).eq.$darkJetID)  is_a_j(i)=.true. ! $darkJetName is a jet
 c-charged-leptons
          if (abs(idup(i,1,iproc)).eq.11)  is_a_l(i)=.true. ! e+  e-
          if (abs(idup(i,1,iproc)).eq.13)  is_a_l(i)=.true. ! mu+ mu-
