diff -ruN MG5_aMC_v2_9_13_orig/madgraph/interface/madevent_interface.py MG5_aMC_v2_9_13/madgraph/interface/madevent_interface.py
--- MG5_aMC_v2_9_13_orig/madgraph/interface/madevent_interface.py	2022-12-16 02:05:07.000000000 -0600
+++ MG5_aMC_v2_9_13/madgraph/interface/madevent_interface.py	2023-10-03 04:48:34.205435475 -0500
@@ -2491,7 +2491,7 @@
         if self.run_card['gridpack'] in self.true:        
             # Running gridpack warmup
             gridpack_opts=[('accuracy', 0.01),
-                           ('points', 2000),
+                           ('points', os.getenv("GRIDPACK_NEVENTS") if "GRIDPACK_NEVENTS" in os.environ else 2000),
                            ('iterations',8),
                            ('gridpack','.true.')]
             logger.info('Generating gridpack with run name %s' % self.run_name)
